Cargo.toml

```diff
@@ -30,10 +30,10 @@ futures = "0.3.30"
 futures-core = "0.3.30"
 hex = "0.4.3"
 log = "0.4.20"
-openmls = { git = "https://github.com/xmtp/openmls", rev = "0da7dcb" }
-openmls_basic_credential = { git = "https://github.com/xmtp/openmls", rev = "0da7dcb" }
-openmls_rust_crypto = { git = "https://github.com/xmtp/openmls", rev = "0da7dcb" }
-openmls_traits = { git = "https://github.com/xmtp/openmls", rev = "0da7dcb" }
+openmls = { git = "https://github.com/Bren2010/openmls", rev = "3992d09" }
+openmls_basic_credential = { git = "https://github.com/Bren2010/openmls", rev = "3992d09" }
+openmls_rust_crypto = { git = "https://github.com/Bren2010/openmls", rev = "3992d09" }
+openmls_traits = { git = "https://github.com/Bren2010/openmls", rev = "3992d09" }
 prost = "^0.12"
```

Comment:
Shouldn't a PR be put up against https://github.com/xmtp/openmls with any necessary changes here?  
(Rather than changing the base repo on which we're using. )
Without more context here for why this change is necessary, I would advise on holding off on merging this until:
- It's clear here why we should switch to using Brendan's fork, or
- we wait for the changes to land in `xmtp/openmls` and we retain that dependency.

cc @neekolas @richardhuaaa

Reply:
There is a PR here: https://github.com/xmtp/openmls/pull/20

The Cargo.toml here was just changed temporarily